src/ast/Clause.cpp

```diff
@@ -49,6 +49,10 @@ void Clause::setBodyLiterals(VecOwn<Literal> body) {
     bodyLiterals = std::move(body);
 }
 
+QualifiedName const& Clause::getQualifiedName() const {
```

Comment:
Can we make a utility function for this? I would prefer that the AST nodes are PODS.